class.jetpack-twitter-cards.php

```diff
@@ -36,6 +36,18 @@ static function twitter_cards_tags( $og_tags ) {
 		}
 
 		if ( ! is_singular() || ! empty( $og_tags['twitter:card'] ) ) {
+			/**
+			 * Filter the default Twitter card image, used when no image can be found in a post.
+			 *
```

Comment:
Could you add `@module sharedaddy` here, so our codex parser can pick the filter up?